op-batcher/batcher/channel_builder.go

```diff
@@ -298,9 +304,9 @@ func (c *ChannelBuilder) updateTimeout(timeoutBlockNum uint64, reason error) {
 	}
 }
 
-// checkTimeout checks if the channel is timed out at the given block number and
+// CheckTimeout checks if the channel is timed out at the given block number and
 // in this case marks the channel as full, if it wasn't full already.
-func (c *ChannelBuilder) checkTimeout(blockNum uint64) {
+func (c *ChannelBuilder) CheckTimeout(blockNum uint64) {
```

Comment:
maybe we take this opportunity to say that it's a L1 block number, now that it lands in the public api of the `ChannelBuilder`
```suggestion
func (c *ChannelBuilder) CheckTimeout(l1BlockNum uint64) {
```